lib/Auth/Source/OAuth2.php

```diff
@@ -213,6 +204,45 @@ function () use ($provider, $accessToken) {
         );
 
         $attributes = $resourceOwner->toArray();
+
+        if (
```

Comment:
why did this block move here, and have the `logIdTokenJson` condition removed?

Reply:
Was for easier debugging, I put it back. Sorry for inconvenience.